hoot-services/src/test/java/hoot/services/controllers/ingest/ImportCommandTest.java

```diff
@@ -33,6 +33,7 @@
 import static hoot.services.controllers.ingest.UploadClassification.*;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
 
 import java.io.File;
```

Comment:
Maybe also there could be a test for multiple options in the `advUploadOpts` string?